lms/envs/production.py

```diff
@@ -1138,10 +1138,16 @@
 )
 
 # Custom urls in navbar
+COLARAZ_SUB_DOMAIN = ENV_TOKENS.get(
+    'COLARAZ_SUB_DOMAIN',
+    COLARAZ_SUB_DOMAIN
+)
 COLARAZ_NAVIGATION_URLS = ENV_TOKENS.get(
     'COLARAZ_NAVIGATION_URLS'
 )
-
+COLARAZ_APP_LINKS_API = ENV_TOKENS.get(
+    'COLARAZ_APP_LINKS_API'
```

Comment:
Don't we have any default value for links that we can add in `common.py`?

Reply:
I think it should give an error in case not set appropriately. because its an api_endpoint which can change easily.